qa/tasks/cephadm.py

```diff
@@ -92,19 +93,13 @@ def download_cephadm(ctx, config, ref):
         if git_url.startswith('https://github.com/'):
             # git archive doesn't like https:// URLs, which we use with github.
             rest = git_url.split('https://github.com/', 1)[1]
-            rest.replace('.git/', '/')  # no .git suffix
+            rest = re.sub(r'(\.git$)|(\.git/$)', '', rest) # no .git suffix
```

Comment:
nit: more compact regex is `r'\.git/?$'`

Reply:
thanks for reminding me of this syntax